GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+* Coordinate with the CNCF regarding usage of the Prometheus brand and how that brand can be used in relation to other efforts or vendors.
+* Receive and handle reports about code of conduct violations and maintain confidentiality.
+* Act as the final escalation point and decide for any disputes, issues, clarifications, or escalations within the project scope.
+* Have “owners” role for the `prometheus` and `prometheus-community` github organizations.
+* Ultimate responsibility for all key project functions, unless delegated to a team. For example, vulnerability and security report handling, or github administration.
+
+## Committee Meetings
+
+The Steering Committee meets monthly, or as-needed.  
+Meetings are held online, and are public by default (anyone can attend).
+
+Given the private nature of some of these discussions (e.g. privacy, private emails to the committee, code of conduct violations, escalations, disputes between members, security reports, etc.) some meetings are held in private.
+
+Public meeting recordings and notes are public and private meeting notes are private.
+
+Questions and proposals for changes to governance or community procedures are posted as [issues in the governance repo](https://github.com/prometheus/governance/issues), and the Steering Committee invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their employer.
+
+The current membership of the committee is currently (listed alphabetically by first name):
+
+| \\&nbsp;                                                         | Member           | Organization |  
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+## Decision process
+
+The Steering Committee defaults to deciding by consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing  
+written policy, creating, removing, or modifying a working group, all spending,  
+hiring, and contracting, official responses to publicly raised issues, or any  
+other decisions that at least two of the members present decide require a vote.
+
+Votes happen by simple majority.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. The Steering Committee can be reached privately by emailing [prometheus-steering@googlegroups.com](mailto:prometheus-steering@googlegroups.com). This is a private discussion list to which all members of the committee have access.
+2. The Steering Committee can be reached publicly by opening an issue on [the governance repository](https://github.com/prometheus/governance/issues) and indicating that you would like attention from the steering committee.
+
+## Composition
+
+The steering committee has 7 seats. These seats are open to any project member. See [Candidate Eligibility](#candidate-eligibility) for a definition.
```

Comment:
```suggestion
The Steering Committee has 5 seats. These seats are open to any project member. See [Candidate Eligibility](#candidate-eligibility) for a definition.
```

At 51 members, ~1/7th seems large; ~1/10th seems better.

Reply:
I would also prefer a smaller steering committee, but not for this reason.

I expect the project to have _way_ more than 51 members very soon. There are 170 individuals with 20+ contributions over the last year (according to CNCF dev-stats).